src/jvmTest/kotlin/fr/acinq/lightning/db/sqlite/SqliteChannelsDb.kt

```diff
@@ -51,20 +52,21 @@ class SqliteChannelsDb(val nodeParams: NodeParams, val sqlite: Connection) : Cha
     }
 
     override suspend fun addOrUpdateChannel(state: ChannelStateWithCommitments) {
-        withContext(Dispatchers.IO) {
-            val data = Serialization.serialize(state)
-            using(sqlite.prepareStatement("UPDATE local_channels SET data=? WHERE channel_id=?")) { update ->
-                update.setBytes(1, data)
-                update.setBytes(2, state.channelId.toByteArray())
-                if (update.executeUpdate() == 0) {
-                    using(sqlite.prepareStatement("INSERT INTO local_channels VALUES (?, ?, 0)")) { statement ->
-                        statement.setBytes(1, state.channelId.toByteArray())
-                        statement.setBytes(2, data)
-                        statement.executeUpdate()
-                    }
-                }
-            }
-        }
+        error("not implemented")
+//        withContext(Dispatchers.IO) {
+//            val data = Serialization.serialize(null as ChannelContext, state)
+//            using(sqlite.prepareStatement("UPDATE local_channels SET data=? WHERE channel_id=?")) { update ->
+//                update.setBytes(1, data)
+//                update.setBytes(2, state.channelId.toByteArray())
+//                if (update.executeUpdate() == 0) {
+//                    using(sqlite.prepareStatement("INSERT INTO local_channels VALUES (?, ?, 0)")) { statement ->
+//                        statement.setBytes(1, state.channelId.toByteArray())
+//                        statement.setBytes(2, data)
+//                        statement.executeUpdate()
+//                    }
+//                }
+//            }
+//        }
```

Comment:
Why is that commented out?

Reply:
How are the tests passing? 😱 

Also, this should take `PersistendChannelState`

Reply:
Done in c5365f4.